src/citrus/reconciler.cljs

```diff
@@ -14,13 +15,53 @@
     (release-fn id))
   (vreset! scheduled? (schedule-fn f)))
 
+(defn citrus-default-handler
+  "Implements Citrus' default event handling (as of 3.2.3).
+
+  This function can be copied into your project and adapted to your needs.
+
+  `events` is expected to be a list of events (tuples):
+
+     [ctrl event-key event-args]"
+  [reconciler events]
+  (let [controllers (gobj/get reconciler "controllers")
```

Comment:
I think that properties of `Reconciler` instance should be accessed via interop syntax `(.-controllers reconciler)`, so when compiled with ADVANCED those properties could be renamed consistently across codebase.

Reply:
Good point. I initially tried to use these getters instead of property access because shadow-cljs logged some warnings and I was trying to understand why something else didn’t work. Will revert back to interop getters 👍

Reply:
bafab61